boards/adafruit-clue/Makefile.dep

```diff
@@ -0,0 +1,27 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+  USEMODULE += apds9960
+  USEMODULE += bmp280_i2c
+  USEMODULE += lis3mdl
+  USEMODULE += sht3x
+endif
+
+# Default auto-initialization of usbus for stdio other USB
+DEFAULT_MODULE += auto_init_usbus
+
+ifneq (,$(filter auto_init_usbus,$(USEMODULE)))
+  # Use stdio over USB
+  USEMODULE += stdio_cdc_acm
+
+  # This boards requires support for nrfutil bootloader.
+  FEATURES_REQUIRED += bootloader_nrfutil
+
+  # and for the board reset via USB mechanism
+  USEMODULE += usb_board_reset
+endif
```

Comment:
`auto_init` modules can not trigger dependency

Reply:
why ?

Reply:
Maybe its better to say they shouldn't, #13651.